pkg/controller/gitrepo/gitrepo_reconcile.go

```diff
@@ -137,7 +137,12 @@ func (r *ReconcileGitRepo) Reconcile(request reconcile.Request) (reconcile.Resul
 			reqLogger.Info("keys differed from CRD, keys re-applied to repository")
 		}
 
-		helpers.AddTenantLabel(&instance.ObjectMeta, instance.Spec.TenantRef.Name)
```

Comment:
The tenant label is added on line 46: https://github.com/projectsyn/lieutenant-operator/pull/50/files#diff-0665f603134a69d730eba6befa1225f9R46